code-studio/src/js/pd/workshop_dashboard/attendance/session_attendance.jsx

```diff
@@ -1,6 +1,10 @@
 /* global React */
 
-var SessionAttendanceRow = require('./session_attendance_row.jsx');
+/*
+  Display and edit attendance for a workshop session, for display in a WorkshopAttendance tab.
+ */
```

Comment:
Nit: No need to change, but in general our style for multi-line comments in JS is to prepend each line with an asterisk, ie.
```
/**
 *  Display and edit attendance for a workshop session, for display in a WorkshopAttendance tab.
 */